<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>4</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RPCS3</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLineEdit" name="mw_searchbar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Search...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="sizeSliderContainer" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="sizeSliderContainer_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>14</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>14</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSlider" name="sizeSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="bootInstallPkgAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGame_List_Icons">
     <property name="title">
      <string>Game List Icons</string>
     </property>
     <addaction name="setIconSizeTinyAct"/>
     <addaction name="setIconSizeSmallAct"/>
     <addaction name="setIconSizeMediumAct"/>
     <addaction name="setIconSizeLargeAct"/>
    </widget>
    <widget class="QMenu" name="menuGame_List_Mode">
     <property name="title">
      <string>Game List Mode</string>
     </property>
     <addaction name="setlistModeListAct"/>
     <addaction name="setlistModeGridAct"/>
    </widget>
    <addaction name="showGameListAct"/>
    <addaction name="separator"/>
    <addaction name="refreshGameListAct"/>
    <addaction name="menuGame_List_Mode"/>
    <addaction name="menuGame_List_Icons"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="bootInstallPkgAct">
   <property name="text">
    <string>Install Packages (PKG)</string>
   </property>
   <property name="toolTip">
    <string>Install application from a .pkg file</string>
   </property>
  </action>
  <action name="bootInstallPupAct">
   <property name="text">
    <string>Install Firmware</string>
   </property>
   <property name="toolTip">
    <string>Install firmware from PS3UPDAT.PUP</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit RPCS3</string>
   </property>
   <property name="statusTip">
    <string>Exit the application.</string>
   </property>
  </action>
  <action name="showGameListAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Game List</string>
   </property>
  </action>
  <action name="refreshGameListAct">
   <property name="text">
    <string>Game List Refresh</string>
   </property>
  </action>
  <action name="setIconSizeTinyAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tiny</string>
   </property>
  </action>
  <action name="setIconSizeSmallAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="setIconSizeMediumAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="setIconSizeLargeAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large</string>
   </property>
  </action>
  <action name="setlistModeListAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>List View</string>
   </property>
  </action>
  <action name="setlistModeGridAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../rpcs3/rpcs3/resources.qrc"/>
 </resources>
 <connections/>
</ui>
